encoder/vaapiencoder_vp9.h

```diff
@@ -0,0 +1,79 @@
+/*
+ * Copyright (C) 2014 Intel Corporation. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef vaapiencoder_vp9_h
+#define vaapiencoder_vp9_h
+
+#include "vaapiencoder_base.h"
+#include "vaapi/vaapiptrs.h"
+#include <va/va_enc_vp9.h>
+#include <deque>
+
+namespace YamiMediaCodec {
+
+class VaapiEncPictureVP9;
+class VaapiEncoderVP9 : public VaapiEncoderBase {
+public:
+    typedef SharedPtr<VaapiEncPictureVP9> PicturePtr;
+    typedef SurfacePtr ReferencePtr;
+
+    VaapiEncoderVP9();
+    ~VaapiEncoderVP9();
+    virtual YamiStatus start();
+    virtual void flush();
+    virtual YamiStatus stop();
+
+    virtual YamiStatus getParameters(VideoParamConfigType type, Yami_PTR);
+    virtual YamiStatus setParameters(VideoParamConfigType type, Yami_PTR);
+    virtual YamiStatus getMaxOutSize(uint32_t* maxSize);
+
+protected:
+    virtual YamiStatus doEncode(const SurfacePtr&, uint64_t timeStamp,
+                                bool forceKeyFrame = false);
```

Comment:
I applied clang format on all the new files, it did find places needed to modify, this wasn't one of those.